apps/system/js/quick_settings.js

```diff
@@ -236,7 +236,8 @@ var QuickSettings = {
 
           case this.bluetooth:
             // do nothing if bluetooth isn't ready
-            if (this.bluetooth.dataset.initializing)
+            if (this.bluetooth.dataset.initializing 
+                || this.airplaneMode.dataset.enabling)
```

Comment:
Let's also early return when `this.airplaneMode.dataset.disabling`. And the problem should also exist for wifi, so I would suggest to add the same check for it.